debian/changelog

```diff
@@ -1,3 +1,15 @@
+system76-firmware (1.0.50) jammy; urgency=medium
+
+  * Add pang12
+
+ -- Levi Portenier <levi@system76.com>  Fri, 06 Jan 2023 16:11:55 -0700
+
+system76-firmware (1.0.49) jammy; urgency=medium
```

Comment:
This has the mira b4 in the changelog :grimacing: 

Can we move this into https://github.com/pop-os/system76-firmware/pull/105

Reply:
Or they could be the same PR, and the change log could justbe one entry that  says add pang12 and mira b4